src/dlgButtonSelect.cpp

```diff
@@ -0,0 +1,233 @@
+/***************************************************************************
+ *   Copyright (C) 2008-2013 by Heiko Koehn - KoehnHeiko@googlemail.com    *
+ *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2016 by Stephen Lyons - slysven@virginmedia.com         *
+ *   Copyright (C) 2017 by Ian Adkins - ieadkins@gmail.com                 *
```

Comment:
I would say the original copyrights still apply. Copyright would have no meaning then if you could copy something and strip the original copyrights off.